cmd/powd/main.go

```diff
@@ -276,12 +267,29 @@ func setupLogging(repoPath string) error {
 		"user-service",
 	}
 
-	// powd registered loggers get info level by default.
-	for _, l := range loggers {
-		if err := logging.SetLogLevel(l, "info"); err != nil {
-			return fmt.Errorf("setting up logger %s: %s", l, err)
+	ipfslog := config.GetBool("ipfslog")
+	// If ipfslog is defined, the output format, destination and level
+	// can be managed via native to ipfs/go-log library
+	// environment variables (e.g. GOLOG_LOG_*)
+	if !ipfslog {
```

Comment:
@kalambet , what do you think of removing the needed new `ipfslog` flag, and deriving the behaviour by taking a look at the environment variables. If any `GOLOG_LOG_` is present, maybe we can assume the `ipfslog=true` behaviour.

Doing something like:
```
var ipfslog bool
for _, e := range os.Environ() {
	if strings.HasPrefix(e, "GOLOG_LOG_") {
		ipfslog = true
		break
	}
}
if !ipfslog { 
...
```
Maybe this would avoid some config burden. If the user is providing `GOLOG_LOG_*`, then most probably always would want the behaviour automatically without extra Powergate flags.

Reply:
nice! I'll update the PR. thank you!